### DISCUSSION TITLE
question - webpack/rollup babel with operator overload plugin

### DISCUSSION THREAD

**Discussion by cjbd** [Hedgehog-Computing/hedgehog-lab#132] [General]
hello, i'm studying your project, i'm also interested in using `babel` to add some language feature like `operator overload` to typescript/javascript, here's my sample project repo: https://github.com/cjbd/TypeScript-Babel-Overload , it's basically your `core/transpiler` code

i can use webpack or rollup to create a bundle, but it cannot run, there're loads of errors, i don't understand where i did it wrong, can you please help, thank you!

**Comment by cjbd** [comment]
hello, i'm pretty desperate here, i looked through your code, and build it myself, the lab can build and run
but i only needed the transpiler part, i tried to follow the same by using `react-app-rewired`, but there's always error too, it seems the node.js modules like `fs` is not polyfilled

i thought the `hedgehog-core` is a seperate package, but it only compile with typescript, the webpack config file for the core package is not used at all

can someone please clarify how this project bundled the transpiler into a single file? thanks!

**Comment by himself65** [comment]
could you please show me your code?
